README.md

```diff
@@ -46,6 +46,10 @@ Message or string translation is the conversion of adhoc strings used throughout
 
     {{ 'Hello :name!'|_({ name: 'Friend' }) }}
 
+Translated content is escaped. If you want to use HTML content in your translated string, you can use the `raw` filter.
```

Comment:
@mjauvin did you verify that this works?

Reply:
I just tested this and it does not work... there is a typo, should have been:
```
{{ 'Title:  :title'| _({title: "E = MC<sup>2</sup>"})|raw }}
```

But still does not work...

Reply:
That's because the escaping is done in the filter itself. We could potentially take an optional second argument (default true) to escape the output

Reply:
The method already has MANY arguments, though:
```
    public static function trans($messageId, $params = [], $locale = null)
```